browser/src/control/jsdialog/Widget.ScrolledWindow.js

```diff
@@ -98,43 +100,66 @@ function _scrolledWindowControl(parentContainer, data, builder) {
 				setTimeout(updateSize, 100);
 			return;
 		}
+		
+		if (!noVertical && data.vertical.upper > 0) {
+			if (rowHeight == 0) {
+				// determine the height a row
+				rowHeight = content.clientHeight / Math.min(data.vertical.upper, data.vertical.page_size);
+				// the content height might not include a new row being added, take it into account with the -1
+				if (!Number.isInteger(rowHeight)) {
+					rowHeight = content.clientHeight / (Math.min(data.vertical.upper, data.vertical.page_size) - 1);
+				}
+			}
+			var viewHeight = data.vertical.page_size * rowHeight;
+			var totalContentHeight = (data.vertical.upper -1) * rowHeight;
+
+			if (totalContentHeight != scrollwindow.scrollHeight) {
+				// only if view has changed
+				var marginTop = data.vertical.value * rowHeight;
 
-		if (!noVertical) {
-			content.style.height = (realContentHeight + verticalSteps) + 'px';
-			scrollwindow.style.height = (realContentHeight + margin) + 'px';
+				content.style.marginTop = marginTop + 'px';
+				content.style.height = (totalContentHeight - marginTop) + 'px';
+				scrollwindow.style.height = viewHeight + 'px';
+				scrollwindow.scrollTop = marginTop;
+			}
 		}
 		if (!noHorizontal) {
 			content.style.width = (realContentWidth + horizontalSteps) + 'px';
 			scrollwindow.style.width = (realContentWidth + margin) + 'px';
 		}
 
-		content.scrollTop = data.vertical.value * 10;
 		content.scrollLeft = data.horizontal.value * 10;
-
-		content.style.margin = content.scrollTop + 'px ' + margin + 'px ' + margin + 'px ' + content.scrollLeft + 'px';
+		content.style.marginRight = margin + 'px';
+		content.style.marginLeft = content.scrollLeft + 'px';
```

Comment:
we should try to avoid margin LEFT RIGHT but use instead START END values, so it works also in RTL UI. With `&lang=ar` in the URL